GeeksCoreLibrary/Components/OrderProcess/Services/OrderProcessesService.cs

```diff
@@ -884,7 +884,8 @@ public async Task<PaymentRequestResult> HandlePaymentRequestAsync(IOrderProcesse
                 var basketSettings = await shoppingBasketsService.GetSettingsAsync();
                 foreach (var (main, lines) in shoppingBaskets)
                 {
-                    if (paymentMethodSettings.PaymentServiceProvider.Type != PaymentServiceProviders.NoPsp && basketToConceptOrderMethod == OrderProcessBasketToConceptOrderMethods.Convert)
+                    var copyBasketToConceptOrderOnPspPayment  = (await objectsService.FindSystemObjectByDomainNameAsync("psp_copy_basket_to_concept_order","1")) == "1";
+                    if (paymentMethodSettings.PaymentServiceProvider.Type != PaymentServiceProviders.NoPsp && basketToConceptOrderMethod == OrderProcessBasketToConceptOrderMethods.Convert && copyBasketToConceptOrderOnPspPayment)
```

Comment:
This code is from a pull request from Rinus (https://github.com/happy-geeks/geeks-core-library/pull/548), why is it also in yours? I denied that PR from Rinus, so please remove that code here as well.